exporter/ibft/reader.go

```diff
@@ -69,7 +69,7 @@ func (r *reader) Sync() error {
 		time.Sleep(1 * time.Second)
 	}()
 	netWaitGroup.Wait()
-	hs := ibftsync.NewHistorySync(r.logger, validatorPubkey, r.network, r.storage, r.validateDecidedMsg)
+	hs := ibftsync.NewHistorySync(r.logger, nil, validatorPubkey, r.network, r.storage, r.validateDecidedMsg) // TODO need to pass identifier
```

Comment:
@nivBlox we should pass identifier and not pub key

Reply:
@amir-blox will add the appropriate identifier

Reply:
but we are still passing validatorPK

Reply:
we need to pass the pubkey for the getTopic func

Reply:
@nivBlox please open an issue for it

Reply:
https://github.com/bloxapp/ssv/issues/138